app/models/uphold_connection.rb

```diff
@@ -108,17 +123,41 @@ def uphold_processing?
 
   def can_create_uphold_cards?
     uphold_verified? &&
-      wallet.present? &&
-      wallet.authorized? &&
-      wallet.scope &&
-      wallet.scope.include?("cards:write") &&
+      uphold_access_parameters.present? &&
+      scope.include?("cards:write") &&
+      status != "pending" &&
+      status != "blocked" &&
```

Comment:
I believe we can use `UpholdConnection::BLOCKED` or perhaps expand it to contain all states in https://uphold.com/en/developer/api/documentation/#user-object

Reply:
Good idea!
https://github.com/brave-intl/publishers/pull/2062/commits/58056406d5c24383360219451d242f01b329dcfb